config/main.py

```diff
@@ -1058,7 +1075,11 @@ def add(ctx, interface_name, ip_addr):
             ctx.fail("'interface_name' is None!")
 
     try:
-        ipaddress.ip_network(unicode(ip_addr), strict=False)
+        ip_network = ipaddress.ip_network(unicode(ip_addr), strict=False)
+        if not interface_name.startswith("Loopback") and ip_network.prefixlen == ip_network.max_prefixlen:
+            ctx.fail("Bad mask /{} for 'ip_addr'".format(ip_network.max_prefixlen))
+        if is_ipaddress_overlapped(ip_addr):
+            ctx.fail("'ip_addr' overlaps with existing subnet")
```

Comment:
Suggest printing the IP address in the failure message.

E.g., `ctx.fail("IP address {} overlaps with existing subnet".format(ip_addr))`

Reply:
Done